input/posts/2020-07-08-grpc-reflection-in-net.md

```diff
@@ -0,0 +1,226 @@
+Title: gRPC Server Reflection in the .NET world
+Published: 2020-07-08
+Tags: 
+- .NET
+- gRPC
+- Protocol Buffers
+---
+> gRPC Server Reflection is an optional extension for servers to assist clients in runtime construction of requests without having stub information precompiled into the client.
+> 
+> -- <cite>[gRPC Server Reflection Protocol](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md)</cite>
+
+The primary use case for gRPC Server Reflection is debugging and testing tools. Think cURL or Postman, but for gRPC. There's however another interesting use case, especially in the .NET world where we have code-first support through [protobuf-net.Grpc](https://github.com/protobuf-net/protobuf-net.Grpc).
```

Comment:
I'd reverse the word order in the second sentence: "However there's another interesting..."